.env

```diff
@@ -88,6 +90,8 @@ PHRASEANET_SMTP_AUTH_ENABLED=false
 PHRASEANET_SMTP_SECURE_MODE=null
 PHRASEANET_SMTP_USER=
 PHRASEANET_SMTP_PASSWORD=
+# Allow settings of email parameters inside env file (0/1)
+ENV_SET_PHRASEANET_EMAIL_SETUP=1
```

Comment:
We don't need of this env
check the test in entrypoint.sh 
just set  PHRASEANET_SMTP_ENABLED = false